src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <section class="card__image">
+      </section>
+
+      <section class="card__name card__name--text_medium">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </section>
+
+      <section class="card__code card__code--text_small">
+        Product code : 195434
+      </section>
```

Comment:
no, section is not for this
https://www.geeksforgeeks.org/html-section-tag/#:~:text=The%20section%20tag%20is%20used,to%20both%20browser%20and%20developer.

Reply:
Changed all the section tags with div.
Read from the provided link and understood the correct usage of section tag.